src/js/d3ViewComponent.js

```diff
@@ -17,6 +17,8 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
     // 1. Convert jQuery DOM elements to D3 elements;
     // 2. Attach D3 DOM event listeners;
     // 3. Synthesize that.options.styles and that.options.selectors to combine elements with the same key into that.classes
+    // 4. Create a basic SVG drawing area for use by implementing grades that actually
+    // draw charts
 
     fluid.defaults("floe.d3ViewComponent", {
```

Comment:
It would be nice to have a ```d3ViewComponent``` that only contains the fundamental implementation for being a bridge to connect jQuery, D3 and infusion without dealing with the actual selectors and rendering, the same idea of having ```fluid.viewComponent``` in infusion.

I would suggest to keep ```floe.d3ViewComponent``` at the old state and use it to create another component that does the basic SVG drawing.